assets/js/modules/analytics/components/setup/SetupForm.js

```diff
@@ -76,11 +78,14 @@ export default function SetupForm( { finishSetup } ) {
 			}
 			if ( ! error ) {
 				setValues( FORM_SETUP, { autoSubmit: false } );
-				await trackEvent( 'analytics_setup', 'analytics_configured' );
+				await trackEvent(
+					`${ viewContext }_analytics`,
+					'analytics_configured'
+				);
```

Comment:
@felixarntz, do we need to change the event name here? Or we shouldn't change this event at all?

Reply:
@eugene-manuilov @ivankruchkoff Oh I had missed that these events are still around, I thought they had already been removed in another issue. Anyway, this should be removed.

Reply:
Removed in https://github.com/google/site-kit-wp/pull/4321/commits/ebbe2b8ab57ee87215f28f0395da1268fdf667f2